forward-message-sendgrid/README.md

```diff
@@ -1,3 +1,9 @@
+
+### This Repo is now out of date. 
+
+With the release of Node v18, the Node.js ecosystem is migrating over from the old CommonJS (CJS) standard to the newer, ES Modules (ESM) standard. You can read about the differences in far more detail in this [blog Post.](https://redfin.engineering/node-modules-at-war-why-commonjs-and-es-modules-cant-get-along-9617135eeca1). The following snippets may causes errors. 
```

Comment:
If we have this info in the main README as it would be with #501 then I do not think we need it here